region/crateria/east.json

```diff
@@ -814,6 +814,34 @@
                 }
               ]
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "West Ocean: G-Mode Setup Lure Zeb from Below (to Attic)",
+                  "notable": true,
+                  "requires": [
+                    "canTrickyUseFrozenEnemies",
+                    {"or": [
+                      "h_canBombThings",
+                      "h_canUseSpringBall"
+                    ]},
+                    "canCameraManip"
+                  ],
+                  "note": [
+                    "Go down to the Zeb Farm which normally leads to the Bowling Alley Path.",
+                    "While standing on the right of the Zeb farm, freeze the Zeb while it is still moving up and facing to the right.",
+                    "Quickly go through the morph tunnel with Bombs or Spring Ball, and get onto the small floating platform just above the Zeb farm.",
+                    "After the Zeb is visible, freeze it and jump on top of it. Use the Zeb as a platform to climb high enough to get it to be aligned with the door.",
+                    "Use the background to see how high the Zeb is - Get the Zeb 2-3 tiles higher than the thinner bright line in the background.",
+                    "Note that if the Zeb is off screen horizontally it will despawn, but if it's off screen vertically it will not.",
```

Comment:
```suggestion
                    "Note that if the Zeb is unfrozen and off screen horizontally it will despawn, but if it's off screen vertically it will not.",
```